File: src/main/java/com/etour/app/entity/TourMaster.java
package com.etour.app.entity;

import jakarta.persistence.*;

@Entity
@Table(name = "tour_master")
public class TourMaster {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "tour_id", nullable = false)
    private Integer id;

    @ManyToOne(fetch = FetchType.LAZY, optional = false)
    @JoinColumn(name = "catmaster_id", nullable = false)
    private CategoryMaster catmaster;

    @ManyToOne(fetch = FetchType.LAZY, optional = false)
    @JoinColumn(name = "departure_date_id", nullable = false)
    private DepartureDateMaster departureDate;

    @Column(name = "description", length = 500)
    private String description;

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public CategoryMaster getCatmaster() {
        return catmaster;
    }

    public void setCatmaster(CategoryMaster catmaster) {
        this.catmaster = catmaster;
    }

    public DepartureDateMaster getDepartureDate() {
        return departureDate;
    }

    public void setDepartureDate(DepartureDateMaster departureDate) {
        this.departureDate = departureDate;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

}

File: dotnet/backend/Models/tour_master.cs
using System;
using System.Collections.Generic;

namespace Etour_Backend_dotnet.Models;

public partial class tour_master
{
    public int tour_id { get; set; }

    public int catmaster_id { get; set; }

    public int departure_date_id { get; set; }

    public string? description { get; set; }

    public virtual ICollection<booking_header> booking_header { get; set; } = new List<booking_header>();

    public virtual category_master catmaster { get; set; } = null!;

    public virtual departure_date_master departure_date { get; set; } = null!;

    public virtual ICollection<tour_addon_master> tour_addon_master { get; set; } = new List<tour_addon_master>();
}
